src/App.tsx

```diff
@@ -14,6 +14,7 @@ function App() {
   const [ isLogged, setIsLogged ] = useState(false)
   const [ searchInput, setSearchInput ] = useState('')
   const [ filters, setFilters ] = useState<WhereObject[]>([])
+  const [ sorts, setSorts ] = useState<string[]>([])
```

Comment:
Not so important, but 'sorts' is a confusing name. Maybe you can rename it `[ sorting, setSorting ] `